paper_framework_sources/p2079_system_execution_context.md

```diff
@@ -483,9 +481,20 @@ Do we want link-time replaceability or run-time replaceability?
 
 ## Do we want to standardize an ABI for system scheduler (as opposed to leaving this to be implementation defined)?
 
-Proposed answer: NO.
-We want to allow vendors the flexibility of choosing the best implementation.
-At the same time, the authors of this paper are willing to contribute to creating an informal basis for implementing replaceability in a common way among major vendors.
+Proposed answer: YES.
+
+To allow users to easily change on the scheduler, they need to know how to replace the system scheduler backend.
+The best way to do that is if we standardize the interface that the system scheduler has.
+
+Tradeoffs:
+* Pro: allow users to replace the default implementation.
+* Pro: allow users to react faster to new technologies (without needing to wait for a new C++ release cycle).
+  * I.e., similar to adopting io_uring, which got heavy adoption in a short amount of time
+* Pro: increased portability
+* Con: slightly larger interface needed to be standardized, and this is not something common for the C++ standard.
+* Con: May allow vendors less freedom on implementing best interfaces for the targeted platforms.
+* Con: May decrease the consensus of the paper.
```

Comment:
I would remove that completely because from my perspective we should not decide for the committee if something increases or decreases consensus. Otherwise, it looks like a "hint" for decreasing consensus (that we are proposing something bad)
```suggestion

```